slides/risw-first-year-wg-sep2023.qmd

```diff
@@ -0,0 +1,229 @@
+- The [R Validation Hub](https://www.pharmar.org/) is an important working group here
+
+## Relation to the validation topic (cont'd)
+
+- Base R and recommended packages are developed by the R Foundation, developed using best practices.
+- Our working group is spreading the word about best development practices such that contributed packages can achieve the same level of quality, hence aiding subsequent validation of them.
+- And we develop specific R packages with these best practices that can thus be easily validated.
+
+## Members and meetings
+
+```{r calc-stats}
+library(readr)
+library(dplyr)
+members <- read_csv("../data/members.csv") |> filter(SWE_WG_Member == 1)
+n_members <- nrow(members)
+unique_orgs <- members |> pull("Affiliation") |> unique() |> sort()
+```
+
+- Currently `r n_members` members
+  - new members are welcome! (incl. academia/regulators/etc.)
+- Currently `r length(unique_orgs)` organizations
+  - `r paste(unique_orgs, collapse = ", ")`
+- Meet every 2 weeks
+
+## Workstreams
+
+- Mixed Models for Repeated Measures (MMRM) [🌎](https://rconsortium.github.io/asa-biop-swe-wg/mmrm_R_package.html)
+  - Develop `mmrm` (see below) to use frequentist inference in MMRM
+- Bayesian MMRM [🌎](https://rconsortium.github.io/asa-biop-swe-wg/bayesian_mmrm_R_package.html)
+  - Develop `brms.mmrm` (see below)
+- Health Technology Assessment [🌎](https://github.com/hta-pharma/)
+  - Develop open-source R tools to support HTA dossier submission across various countries, particularly the topics with unmet needs in R implementation and/or related to EUnetHTA
+- Note: Also "just" contributing to workstreams is great!
+
+# Achievements in the first year
+
+## New R packages released to CRAN
+
+- `mmrm` 
+  - R package for frequentist inference in MMRM, based on `TMB` (which provides automatic differentiation in `C++` and R frontend)
+  - See documentation [🌎](https://openpharma.github.io/mmrm/)
+  - Easiest to install from CRAN [🌎](https://cran.r-project.org/package=mmrm)
+- `brms.mmrm` 
+  - R package for Bayesian inference in MMRM, based on `brms` (as Stan frontend for HMC sampling)
+  - See documentation [🌎](https://openpharma.github.io/brms.mmrm/)
+  - Easiest to install from CRAN [🌎](https://cran.r-project.org/package=brms.mmrm)
+
+## Why was the MMRM topic important?
+
+- MMRM is a popular analysis method for longitudinal continuous outcomes in randomized clinical trials
+- No tailored R package with sufficient capabilities/reliability
+- Also used as backbone for more recent methods such as multiple imputation
+- We have run comparison analyses with other R packages, namely `nlme`, `glmmTMB` and `lme4` as well as SAS `PROC GLIMMIX` and have found that `mmrm` is the fastest of them and provides closes results to `PROC GLIMMIX`
```

Comment:
Our current vignette compares with SAS `PROC MIXED`, did we also run comparison with `PROC GLIMMIX`?

Reply:
Yeah we heard from @yonicd that this one is newer. Ongoing PR in mmrm updates the comparison to using this

Reply:
Thanks Daniel.